packages/cutler/bin/cutler.dart

```diff
@@ -27,24 +30,60 @@ class Cutler extends CommandRunner<int> {
       ..addFlag('update', defaultsTo: true, help: 'Update checkouts.');
   }
 
+  final Logger _logger;
+
+  Iterable<String> missingDirectories(String rootDir) {
+    return Repo.values.map((repo) => '$rootDir/${repo.path}').where(
+          (path) => !Directory(path).existsSync(),
+        );
+  }
+
+  String fallbackRootDir() {
```

Comment:
Same here re: doc string. What is being fallen back to/from? Why would fallback happen?

Reply:
Added.